src/app/core/services/travelperk/travelperk-advanced-settings.service.ts

```
@@ -0,0 +1,111 @@
+
+  static createFormGroup(data: TravelperkAdvancedSettingArray): FormGroup {
+      return new FormGroup ({
+          destinationName: new FormControl(data.destination_name || '', Validators.required),
+          sourceName: new FormControl(data.source_name || null, Validators.required),
+          isRequired: new FormControl(true)
+      });
+  }
+
+  static constructFormArray(arrayData: TravelperkAdvancedSettingGet, source_fields: TravelperkDestinationAttribuite[]): FormGroup[] {
+      const mappingPayload:FormGroup[] = [];
+      const defaultCategories = this.getDefaultCategory();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)